### PR TITLE
R11PIT-159: Install both .NET Core versions in parallel

### PR DESCRIPTION
**Pull request by pedro132**
* Install both 2.1 and 3.1 .NET Core Windows Server Hosting bundles, to allow SetupTools to be used to install the pre-requisites of all OutSystems 11 versions
* Update 2.1 version downloaded in CreateOfflineBundle.ps1 script to be the same as used in Install-OSServerPreReqs.ps1
* Includes QA

### CODE REVIEW

**Comment by pintonunes** [comment]
Can you fix the test?

**Comment by pedro132** [comment]
Tests are now running successfully :)

File: src/Outsystems.SetupTools/Lib/PlatformSetup.ps1 [hunk 1]
```diff
@@ -431,6 +442,16 @@ function InstallDotNetCore([string]$Sources)
     return $($result.ExitCode)
 }
 
+function InstallDotNetCore([string]$Sources)
+{
+    return InnerInstallDotNetCore -InstallVersion21 $False -Sources $SourcePath
```
Comment (pintonunes) [inline src/Outsystems.SetupTools/Lib/PlatformSetup.ps1:447]: This should be $Sources and not $SourcePath

**Review by pintonunes — CHANGES_REQUESTED** [review]

File: src/Outsystems.SetupTools/Lib/PlatformSetup.ps1 (hunk 1 above)
Comment (pedro132) [reply src/Outsystems.SetupTools/Lib/PlatformSetup.ps1]: Fixed

**Review by pintonunes — APPROVED** [review]

**Pull request merged**